src/ui/.eslintrc.json

```diff
@@ -9,7 +9,7 @@
       "es6": true
     },
     "parserOptions": {
-      "ecmaVersion": 6,
+      "ecmaVersion": 2018,
```

Comment:
It doesn't really matter what this is set to, as long as it's set to something sensible. `ecmaVersion 2018` is very nearly `ecmaVersion 6`. It just adds a few constructs that are more modern. In any case, `babel` compiles to the same version of JavaScript regardless of this setting.